pollen_py/pollen/depth/parse_data.py

```diff
@@ -274,19 +285,19 @@ def config_parser(parser):
 
 def run(args):
     if args.from_verilog or args.from_interp:
-        with open(filename, "r") as fp:
```

Comment:
@susan-garry I'm confused as to how this `filename` thing ever worked. Does `args.filename` do the trick, or is there a deeper issue?